serverless.yml

```diff
@@ -236,3 +233,31 @@ functions:
     package:
       include:
         - functions/AutoRemediateEC-005.js
+    role: revokeSecurityGroupIngressRole
+
+resources:
+
+  Resources:
+
+    revokeSecurityGroupIngressRole:
+      Type: AWS::IAM::Role
+      Properties:
+        RoleName: RevokeSecurityGroupIngressRole
```

Comment:
To keep it consistent, can you please change revokeSecurityGroupIngressRole to AutoRemediateEC-002Role